Mathlib/CategoryTheory/Galois/Decomposition.lean

```diff
@@ -308,6 +308,14 @@ lemma exists_hom_from_galois_of_connected (X : C) [IsConnected X] :
     ∃ (A : C) (_ : A ⟶ X), IsGalois A :=
   exists_hom_from_galois_of_fiber_nonempty F X inferInstance
 
+/-- To check equality of  natural transformations `F ⟶ F`, it suffices to check it on
+Galois objects. -/
+lemma natTrans_ext_of_isGalois {t s : F ⟶ F} (h : ∀ (X : C) [IsGalois X], t.app X = s.app X) :
```

Comment:
It seems to me this could be generalized to natural transformations from `F` to any functor `G`, with the same proof.

Reply:
Indeed, this works. Changed.